about.html

```diff
@@ -0,0 +1,175 @@
+              class="main-logo-img"
+          /></a>
+          <ul class="navbar-ul">
+            <li>
+              <a href="./index.html" class="desktop-nav-link">Main Page</a>
+            </li>
+            <li>
+              <a href="#about-us" class="desktop-nav-link">About Us</a>
+            </li>
+            <li>
+              <a href="#past-activities" class="desktop-nav-link"
+                >Past Activities</a
+              >
+            </li>
+            <li><a href="#contact" class="desktop-nav-link">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
+      <div class="pryContainer">
+        <img
+          src="./images/line-decoration.png"
+          alt="Background image"
+          class="bg-header-image"
+        />
+        <img
+          src="./images/design-pattern.png"
+          alt="Background image"
+          class="bg-header-image-rotate"
+        />
+        <div class="pryContainerContent">
+          <div class="pryContainerHeader flx-col-center">
+            <h2 class="myColor text-center">"Who are we?"</h2>
+            <h1 class="myColor text-center">
+              AIOU ENTREPRENEURSHIP & TECHNOLOGY CONFERENCE
+            </h1>
+          </div>
+          <div class="headerContents text-center margin-top-big">
+            <p>
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Et,
+              illo! Rem cupiditate nemo minima voluptate laboriosam deserunt,
+              dignissimos quia incidunt id pariatur odit laborum sunt
+              reprehenderit labore blanditiis temporibus modi.
+            </p>
+          </div>
+          <div class="infos">
+            <p class="location light-black text-center">
+              <i
+                >Please Contact Us with Email for any further questions about
+                this activity!</i
+              ><br /><b>forazkar@gmail.com</b>
+            </p>
+          </div>
+        </div>
+      </div>
+    </header>
+    <section class="about-us flx-col-center text-center" id="about-us">
+      <h2 class="about-us-head light-black heading-medium margin-top-big">
+        About Us
+      </h2>
+      <hr class="myLine text-center" />
+      <img
+        src="https://securityconference.org/assets/media/e/5/csm_themen_cyber-security-and-technology_header_compr_2037a5dc78.jpg"
+        alt="about us community photo"
+        class="family-photo"
+      />
+      <div class="about-us-content">
+        <p class="about-us-paragraph">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Id mollitia
+          magni enim ducimus repudiandae assumenda fugit dolorum doloribus
+          ratione dolor cum, fugiat consequatur quisquam quos deserunt amet illo
+          recusandae distinctio incidunt tenetur veritatis facilis excepturi
+          eum. Consectetur repellendus sunt, dolorum exercitationem maiores
+          optio architecto voluptates deserunt quia nemo quod magni.
```

Comment:
- [ ] As mentioned before, please reduce size of the image to be consistent with the design and put the text above the image.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/175329563-46c12554-4112-4220-b689-609059a725df.png)  | ![image](https://user-images.githubusercontent.com/53402600/175329614-4d649a13-dd7c-4c4c-a979-d433157842b4.png) |

- [ ] Consider also to add partner section in about page to be consistent with the design.
> Behance design
![image](https://user-images.githubusercontent.com/53402600/175330204-ce2d82d5-ae17-42d7-be05-c09a7d8852b5.png)